pages/settings.js

```diff
@@ -0,0 +1,380 @@
+import { headers } from '@/next.config';
+import settingsStyle from '../styles/Settings.module.css'
+import symbolsStyle from '../styles/Symbols.module.css'
+
+import { useState, useEffect } from 'react';
+
+
+export default function Settings (){
+    const [languageProfile, setLanguageProfile] = useState(null);
+    const [dataBot, setDataBot] = useState(null);
+    const [botToken, setBotToken] = useState('');
+    const [helloMessage, setHelloMessage] = useState('');
+    const [like, setLike] = useState('');
+    const [disLike, setDisLike] = useState('');
+    const [report, setReport] = useState('');
+    const [message, setMessage] = useState('');
+    const [accountSettings, setAccountSettings] = useState('');
+    const [menu, setMenu] = useState('');
+    const [back, setBack] = useState('');
+    const [startSearch, setStartSearch] = useState('');
+    const [editForm, setEditForm] = useState('');
+    const [restartForm, setRestartForm] = useState('');
+    const [editFormField, setEditFormField] = useState('');
+    const [activateAccount, setActivateAccount] = useState('');
+    const [deactivateAccount, setDeactivateAccount] = useState('');
+    const [yes, setYes] = useState('');
+    const [no, setNo] = useState('');
+    const [errorMessage, setErrorMessage] = useState('');
+    
+    useEffect(() => {
+        fetch('http://localhost:3000/api/botSettings/get')
+            .then(response => response.json())
+            .then(data => {
+                setDataBot(data);
+            })
+            .catch(error => {
+                console.error('Error:', error);
+            });
+    }, []);
+
+    useEffect(() => {
+        if (dataBot) {
+            fetch('http://localhost:3000/api/languageProfile/one/1')
```

Comment:
Check this https://www.toolsqa.com/rest-assured/rest-routes/ and follow this approach in creating routes both here in express apps.